<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Search</class>
 <widget class="QDialog" name="Search">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="10" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Введите дату рождения</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QDateEdit" name="dateEdit_edit"/>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QComboBox" name="comboBox_main">
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentText">
        <string>Студенты </string>
       </property>
       <item>
        <property name="text">
         <string>Студенты </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Преподователи </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Предметы и преподователи </string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Факультеты и предметы </string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Введите имя</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="departmen_teachers"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="Patronymic_edit"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="Lname_edit"/>
     </item>
     <item row="9" column="1">
      <widget class="QComboBox" name="comboBox_dep"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Academic_degree</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Ceместр</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Факультет</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="academic_degree_teachers"/>
     </item>
     <item row="8" column="1">
      <widget class="QComboBox" name="comboBox_sem"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Academic_title</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="academic_title_teachers"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="Name_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Department </string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Введите фамилию</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="comboBox_3"/>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Subject</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Введите отчество </string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
